peerblock.json

```diff
@@ -1,18 +1,11 @@
 {
     "homepage": "https://forums.peerblock.com/",
     "description": "An IP filter for Windows 7/10, forked from PeerGuardian 2",
-    "version": "1.1.518",
+    "version": "1.1.691",
     "license": "zlib-acknowledgement",
-    "architecture": {
-        "64bit": {
-            "url": "https://downloads.sourceforge.net/project/peerblock/PeerBlock_r518__x64_Release.zip",
-            "hash": "sha1:6bad36ff269db1ce7ba484b9e5dc919df94b87a4"
-        },
-        "32bit": {
-            "url": "https://downloads.sourceforge.net/project/peerblock/PeerBlock_r518__Win32_Release.zip",
-            "hash": "sha1:7b1c3cc19dc3b782e04896d48902d09606ac2d98"
-        }
-    },
+    "url": "https://github.com/scoopinstaller/binary-mirror/blob/master/peerblock/PeerBlock-Setup_v1.1%2B_r691.exe",
```

Comment:
Should be `https://github.com/scoopinstaller/binary-mirror/raw/master/peerblock/PeerBlock-Setup_v1.1%2B_r691.exe`

Please try installing/uninstalling programs before submitting changes.